contactButtonStyles.json

```diff
@@ -0,0 +1,11 @@
+{
+  "Discord": {
+    "bg": "#7289da",
+    "text": "text-white"
+  },
+  "Instagram": {
+    "bg": "#E138B6",
+    "text": "text-white"
+  },
+  "Email": {}
```

Comment:
Instead of using a json file make it an export from a TS file so that it can be extended with interfaces

Reply:
That's a good idea. The motivation of using JSON was to make it more easily updatable if new contact methods are needed. Maybe it would be just as easy to actually update the source code.